platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -465,21 +465,38 @@ influential.
 
 **Platforms** enable developers and operators to deliver applications and
 services faster by providing and managing common capabilities. Platforms bridge
-between platform users and platform capability providers.
+between platform users and platform capability providers, and are developed and
```

Comment:
Do you think "developed" implies writing code? Would we prefer not to imply that platforms require writing code?

Reply:
```suggestion
between platform users and platform capability providers, and are built and
```

Good point. We can use "built", if that sounds better for you.